guides/writing-addons/intro-tutorial.md

```diff
@@ -290,4 +290,31 @@ Lastly, be sure to provide a few notes about how others can contribute to the pr
 
 ## Advanced addon configuration
```

Comment:
```suggestion
## Common addon configurations
```

Eventually I want to dedicate an entire page to addon configurations, so maybe we include solely the basics here. What do you think?

Reply:
I agree